tensorflow/contrib/learn/python/learn/utils/gc_test.py

```diff
@@ -33,8 +33,9 @@
 def _create_parser(base_dir):
   # create a simple parser that pulls the export_version from the directory.
   def parser(path):
-    match = re.match("^" + compat.as_str_any(base_dir) + "/(\\d+)$",
-                     compat.as_str_any(path.path))
+    # Modify the path object for RegEx match for Windows Paths
+    match = re.match("^" + compat.as_str_any(base_dir).replace('\\','/') + "/(\\d+)$",
+                    compat.as_str_any(path.path).replace('\\','/'))
```

Comment:
This substitution doesn't seem safe on POSIX... what if there is a `\` character in a path? Perhaps you should guard this with an `if os.name == 'nt':` block.